src/components/Blog.jsx

```
@@ -0,0 +1,213 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { motion } from "framer-motion";
+import Container from "./ui/Container";
+import BlogImg1 from "../assets/people.png";
+import GroupPics from "../assets/groupphoto.jpg";
+import NavBar from "./NavBar";
+import Footer from "./Footer";
+import BlogImg2 from "../assets/possible.png";
+import BlogImg3 from "../assets/throwback.png";
+import Container from "./ui/Container";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Container'. Consider to delete it or rename it.

'Container' is defined here:


(lint/suspicious/noRedeclare)